mappings/net/minecraft/client/lookup/IdentifierSearchableContainer.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS doj net/minecraft/client/lookup/IdentifierSearchableContainer
```

Comment:
This name is pretty odd. Something like `IndexedSearchableContainer` or `MappedSearchableContainer` would be more appropriate imo.

Reply:
I think Identifier is fine imo